src/dev_genesis_wasm.erl

```
@@ -71,6 +61,30 @@ delegate_request(Msg, Msg2, Opts) ->
             }}
     end.
 
+
+%% @doc Handle normal compute execution with state persistence (GET method).
+do_compute(Msg, Msg2, Opts) ->
+    % Resolve the `delegated-compute@1.0' device.
+    case hb_ao:resolve(Msg, {as, <<"delegated-compute@1.0">>, Msg2}, Opts) of
+        {ok, Msg3} ->
+            % Resolve the `patch@1.0' device.
+            {ok, Msg4} =
+                hb_ao:resolve(
```

I don't know if we should let it crash here. Maybe also return an error like the first `hb_ao:resolve`?